test/distributed/test_nvshmem.py

```diff
@@ -61,11 +62,19 @@ def test_alloc(self) -> None:
         def foo():
             inp = symm_mem.empty(numel, dtype=dtype, device=self.device)
             symm_mem.rendezvous(inp, group=group_name)
+            self.assertGreater(_SymmetricMemory.num_active_allocations(self.device), 0)
 
         foo()
+        self.assertEqual(_SymmetricMemory.num_active_allocations(self.device), 0)
 
-        out = symm_mem.empty(numel, dtype=dtype, device=self.device)
-        symm_mem.rendezvous(out, group=group_name)
+        t0 = symm_mem.empty(numel, dtype=dtype, device=self.device)
+        symm_mem.rendezvous(t0, group=group_name)
+        t1 = symm_mem.empty(numel, dtype=dtype, device=self.device)
+        symm_mem.rendezvous(t1, group=group_name)
+        self.assertGreater(_SymmetricMemory.num_active_allocations(self.device), 0)
```

Comment:
Same as my question below, what is the actual number, why only check larger than 0?